examples/simple/main.tf

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
The changes that you're making to `main.tf` and the other files should be mirrored across all of the other terratest modules. When I was working on the `terraform-aws-lambda-function` module, I would frequently refer back to the changes that @pjdufour-dds was making to `terraform-aws-sqs-queue` (which at the time was the most up-to-date template, I believe). So maybe a quick notification to everyone who is actively developing in dod-iac/dds that they should now use `terraform-aws-acm-certificate` as the template.

Reply:
Only use the `dod-iac/terraform-module-template` repo. From now on that is the place to see the most up-to-date changes. If one of us figures out a new method somewhere else then we should port it back. You should no longer have to use other modules as a basis for your work.